cartographer_ros/cartographer_ros/assets_writer_main.cc

```diff
@@ -150,7 +150,7 @@ void Run(const string& trajectory_filename, const string& bag_filename,
     rosbag::Bag bag;
     bag.open(bag_filename, rosbag::bagmode::Read);
     rosbag::View view(bag);
-    const ::ros::Time begin_time = view.getBeginTime();
+    const ::builtin_interfaces::msg::Time begin_time = view.getBeginTime();
```

Comment:
Yeah I agree that these files can not be ported given that they rely on missing features. It's just surprising to have a sed on "ros::Time" but not the corresponding one on "ros/time.h". I think they should go together, either both changes or none of them.